pydis_site/templates/home/index.html

```diff
@@ -9,19 +9,6 @@
 {% block content %}
   {% include "base/navbar.html" %}
 
-  <!-- Mobile-only Notice -->
-  <section id="mobile-notice" class="message is-primary is-hidden-tablet">
-    <div class="message-header">
-      <p>100K Member Milestone!</p>
-    </div>
-    <div class="message-body">
-      Thanks to all our members for helping us create this friendly and helpful community!
-      <br><br>
-      As a nice treat, we've created a <a href="{% url 'timeline' %}">Timeline page</a> for people
-      to discover the events that made our community what it is today. Be sure to check it out!
-    </div>
-  </section>
-
```

Comment:
What's the reasoning for this removal?

Reply:
This was discussed in `#dev-contrib` and I think the consensus was that this was supposed to be removed. Feel free to provide alternatives.

Reply:
I'm not too big a fan of pull requests that do one thing + many small other things that are unrelated and should probably go in it's own issue, but this is a small change. Do you have a message link to the discussion on `#dev-contrib`?

Reply:
https://canary.discord.com/channels/267624335836053506/635950537262759947/843912385776975902 it starts from here

Reply:
Thanks!